### PR TITLE
[oh-my-zsh] Download oh-my-zsh for your zsh terminal

### PR DESCRIPTION
**Pull request by debodirno**
- curl to download oh-my-zsh from robbyrussell's repository
- Added "-f" flag to prevent slowness
- Added "-x" flag for helping in debugging

### CODE REVIEW

File: oh-my-zsh/download_oh-my-zsh.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -f -x
```
Comment (ikad95) [inline oh-my-zsh/download_oh-my-zsh.sh:1]: -f -x won't make it any faster

**Review by ikad95 — CHANGES_REQUESTED** [review]

File: oh-my-zsh/download_oh-my-zsh.sh (hunk 1 above)
Comment (debodirno) [reply oh-my-zsh/download_oh-my-zsh.sh:1]: "-f" will!

Comment (debodirno) [reply oh-my-zsh/download_oh-my-zsh.sh:1]: -x is for verbose output.
-f is for forcing not to load BASH_ENV.

Comment (ikad95) [reply oh-my-zsh/download_oh-my-zsh.sh:1]: Won't

**Review by ikad95 — APPROVED** [review]

**Pull request merged**